package.json

```diff
@@ -67,6 +67,7 @@
     "uglify-es": "^3.3.9",
     "uglify-js": "^3.4.9",
     "umd-compat-loader": "^2.1.1",
+    "untildify": "^3.0.3",
```

Comment:
Yes, yes, add more random `npm` libraries 🕺

Reply:
What it does is literally a one-liner, I feel like I am falling to the dark side